pkg/storaged/lvol-tabs.jsx

```diff
@@ -50,15 +50,16 @@ function lvol_rename(lvol) {
     });
 }
 
-function lvol_and_fsys_resize(client, lvol, size, offline, passphrase) {
+function lvol_and_fsys_resize(client, lvol, targetSize, offline, passphrase) {
     var block, crypto, fsys;
     var crypto_overhead;
     var vdo;
     var orig_size = lvol.Size;
+    var size = targetSize | 0;
```

Comment:
That surely is the weirdest syntax for "convert this to an integer" that I have ever seen.. To me it looks like a "binary or with zero", which is in fact what it syntactically is.. it just seems to convert its arguments to ints as a side effect.

I would feel a little better if that would use `parseInt(targetSize)` instead, and handle the `null` and `undefined` cases separately (if we even need to handle that at all) -- e.g. `parseInt(targetSize || 0)`, or  just `parseInt(targetSize)` if we can already assume that we have a number.

Reply:
parseInt parses a string https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt  that's why I ended up using this syntax. But it will work as well. I will replace.

Reply:
It's a number, see `validate` function at SizeSlider

Reply:
I see -- `Math.round()` then? That's probably also more correct than parseInt() or `| 0`, which always truncate. I.e. 3.9 would end up as 3, not 4.

Reply:
The size here is already in bytes and not the value we insert in the UI component. Rounding does not offer much.

Reply:
I see - I was afraid it would be the "9.15" from the "Terabytes" input. Thanks!